gui/src/components/JobList.js

```diff
@@ -46,27 +47,66 @@ const renderBody = (jobs, fetching, error) => {
   }
 }
 
-export const JobList = ({jobs, fetching, error}) => (
-  <Card>
-    <Table>
-      <TableHead>
-        <TableRow>
-          <TableCell>ID</TableCell>
-          <TableCell>Created</TableCell>
-          <TableCell>Initiator</TableCell>
-        </TableRow>
-      </TableHead>
-      <TableBody>
-        {renderBody(jobs, fetching, error)}
-      </TableBody>
-    </Table>
-  </Card>
-)
+const handleChangeRowsPerPage = () => {
+  console.log('handleChangeRowsPerPage')
```

Comment:
Should this be in production code?

Reply:
👍 It requires a handler but it should be a no-op